### PR TITLE
Fixes ushanka hiding hair

### PR DESCRIPTION
**Pull request by Cherno00**
## About The Pull Request

Simply reverts the ushanka to show the player's hair while worn.
image

## How This Contributes To The Skyrat Roleplay Experience

 Lets you see people's beautiful hair while they wear the hat :D

## Changelog

:cl:
fix: ushanka shows hair while worn.
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/hat.dm [hunk 1]
```diff
@@ -168,7 +168,7 @@
 	desc = "Perfect for winter in Siberia, da?"
 	icon_state = "ushankadown"
 	inhand_icon_state = "ushankadown"
-	flags_inv = HIDEEARS|HIDEHAIR
+	flags_inv = HIDEEARS
```
Comment (ghost) [inline code/modules/clothing/head/hat.dm]:
```suggestion
	flags_inv = HIDEEARS //SKYRAT EDIT (Original: HIDEEARS|HIDEHAIR)
```
Non-Modular changes (to files not in our modular_skyrat folder) need proper labelling


Also, if you havent, be sure to test this on a private server to make sure it doesnt have unintended jankyness with the hair and hat sprites fighting. Unlikely, but better safe than sorry

**Comment by GoldenAlpharex** [comment]
Don't mind the integration test, I'll fix that part tonight. However, have you tested this to be sure that it works how you intended it to?

**Comment by Gandalf2k15** [comment]
Why would this not hide hair lol

**Comment by GoldenAlpharex** [comment]
> Why would this not hide hair lol

Because winter hats don't typically cover all of your hair. I speak from experience as a Canadian, lol.

**Comment by Cherno00** [comment]
> > Why would this not hide hair lol
> 
> Because winter hats don't typically cover all of your hair. I speak from experience as a Canadian, lol.

**Comment by Cherno00** [comment]
Oops, accidentally hit a button aaaaa

**Comment by Cherno00** [comment]
> Don't mind the integration test, I'll fix that part tonight. However, have you tested this to be sure that it works how you intended it to?

I plan on testing it tonight! Dunno HOW to do that, but I'll figure it out.

**Comment by thestubborn** [comment]
> 
> 
> > Don't mind the integration test, I'll fix that part tonight. However, have you tested this to be sure that it works how you intended it to?
> 
> I plan on testing it tonight! Dunno HOW to do that, but I'll figure it out.

its easy
look in the folder where your skyrat is, like, where you modded the files
click build.bat
wait for that to finish
open dream daemon
select trusted
click the three ... and select skyrat's...tg thingy

**Comment by Cherno00** [comment]
> > > Don't mind the integration test, I'll fix that part tonight. However, have you tested this to be sure that it works how you intended it to?
> > 
> > 
> > I plan on testing it tonight! Dunno HOW to do that, but I'll figure it out.
> 
> its easy look in the folder where your skyrat is, like, where you modded the files click build.bat wait for that to finish open dream daemon select trusted click the three ... and select skyrat's...tg thingy

Tested it! Works perfect <3

**Comment by Cherno00** [comment]
> Don't mind the integration test, I'll fix that part tonight. However, have you tested this to be sure that it works how you intended it to?

Just tested it now. Works fine <3

**Comment by Gandalf2k15** [comment]
Also this is a change not a fix

**Comment by Cherno00** [comment]
Posted image of it working

**Pull request merged**